### PR TITLE
Update RBAC and Teams docs

### PR DESCRIPTION
**Pull request by potocnyj**
We're getting ready to release this to everyone, go through the documentation and cleanup things that are out of date.

### CODE REVIEW

**Comment by potocnyj** [comment]
@gkristic @mtrias @Preetam this is round 1 but I think there's more to do here. PTAL and let me know what you think 👍

File: content/general-reference/rbac.md [hunk 1]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
 weight: "7"
 ---
 
-VividCortex allows restricting access to the systems only to authorized users based
-on roles assigned to teams. This approach is generally referred as
+For enterprise level organizations, VividCortex allows restricting access to the systems only
```
Comment (Preetam) [inline content/general-reference/rbac.md]: `VividCortex allows restricting access to the systems only to authorized users based on roles assigned to teams` sounds weird.

File: content/general-reference/rbac.md [hunk 2]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
...
+to authorized users based on roles assigned to teams. This approach is generally referred as
 [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control).
```
Comment (Preetam) [inline content/general-reference/rbac.md]: This should be `Role-based access control (RBAC)` if it's the first occurrence of "RBAC." You can refer to it as `RBAC` afterwards.

File: content/general-reference/rbac.md [hunk 3]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
...
 
-Before talking more about RBAC, if you aren't familiar with how _Environments_ and _Teams_ work,
+Before talking more about RBAC, if you aren't familiar with how *Environments* and *Teams* work,
```
Comment (Preetam) [inline content/general-reference/rbac.md]: I'd drop `Before talking more about RBAC,`.

File: content/general-reference/rbac.md [hunk 4]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
...
 please make sure you check out the
 [Environments & Teams documentation](/general-reference/environments-team/).
 
 
 ## Roles
 
-A _Role_ is a set of actions that are granted to a user in a context. For example,
-a role of _Read Only_ in the "Production" _environment_ grants read access to all
+A *Role* is a set of actions that are granted to a user in a context. For example,
```
Comment (Preetam) [inline content/general-reference/rbac.md]: A Role *is* granted to a user, so I'd write `A *Role* is a set of actions that is granted to a user`.

File: content/general-reference/rbac.md [hunk 5]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
...
+a role of *Read-Only* in the "Production" *environment* grants read access to all
```
Comment (Preetam) [inline content/general-reference/rbac.md]: I don't think `*environment*` needs to be italicized.

File: content/general-reference/rbac.md [hunk 6]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
...
 the non-sensitive data of the environment.
```
Comment (Preetam) [inline content/general-reference/rbac.md]: I personally prefer `read access to all non-sensitive data`. I'm not sure if they're both valid.

File: content/general-reference/rbac.md [hunk 7]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
...
 
 Roles are assigned in two ways:
 
-- Assigning a _role_ to a _team_
-- Assigning a role to an _environment_ - _team_ couple.
+- Assigning a *role* to a *team*
+- Assigning a *role* to an *environment - team* couple.
```
Comment (Preetam) [inline content/general-reference/rbac.md]: `pair` instead of `couple`?

File: content/general-reference/rbac.md [hunk 8]
```diff
@@ -5,27 +5,27 @@ docs: "reference"
...
 
-By assigning a _role_ to a _team_ you are controlling the default privileges on that team.
+By assigning a *role* to a *team* you are controlling the default privileges on that team.
```
Comment (Preetam) [inline content/general-reference/rbac.md]: Comma after `*team*`

File: content/general-reference/rbac.md [hunk 9]
```diff
@@ -39,18 +39,19 @@ to the "staging" environment, but only read access to the "Production" environme
...
+- *Read-Write*
+- *Read-Only*
 
-The _Administrator_ role allows full access.
+The *Administrator* role allows full access.
 
-The _Read-Write_ role allows users to do things like update host credentials, add new hosts,
-etc. and the _Read only_ role restricts the access to only read operations.
+The *Read-Write* role allows users to do things like update host credentials, add new hosts,
+etc. and the *Read-Only* role restricts the access to only read operations.
```
Comment (Preetam) [inline content/general-reference/rbac.md]: I wouldn't use `like` *and* `etc.`. Also, comma after `etc.`.

File: content/general-reference/rbac.md [hunk 10]
```diff
@@ -39,18 +39,19 @@ to the "staging" environment, but only read access to the "Production" environme
...
 
-The _Read Only_ role only grants the privilege of looking at the reports in VividCortex.
-No sensitive information can be accessed though, such as host credentials, Query Samples
-or api tokens. These privileges are reserved for _Read-Write_ and _Administrator_ roles.
+The *Read-Only* role only grants the privilege of looking at the reports in VividCortex.
```
Comment (Preetam) [inline content/general-reference/rbac.md]: I'd write `looking at reports`.

File: content/general-reference/rbac.md [hunk 11]
```diff
@@ -39,18 +39,19 @@ to the "staging" environment, but only read access to the "Production" environme
...
+No sensitive information can be accessed though, such as host credentials, [Query Samples]
+(/general-reference/samples) or api tokens. These privileges are reserved for *Read-Write*
+and *Administrator* roles.
```
Comment (Preetam) [inline content/general-reference/rbac.md]: > No sensitive information—such as host credentials, query samples, and API tokens—can be accessed. These are reserved for...

**Comment by potocnyj** [comment]
@Preetam covered all of the comments above, anything else before I merge this?

**Pull request merged**